### PR TITLE
ethernet: T5284: Update Realtek r8152 + 8168 driver

### PR DESCRIPTION
**Pull request by cuongdt1994**
Add support next-gen

+ Realtek R8111/8168/8169 1Gbps
+ Realtek R8125 2.5Gbps

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://vyos.dev/T5284

## Checklist:

- [ ] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [ ] I have linked this PR to one or more Phabricator Task(s)
- [ ] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by cuongdt1994** [comment]
image

Tested with vyos-1.4-rolling-202306131810-amd64

File: data/live-build-config/includes.chroot/etc/udev/rules.d/50-usb-realtek-net.rules [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# TP-LINK
+ATTR{idVendor}=="2357", ATTR{idProduct}=="0601", ATTR{bConfigurationValue}!="$env{REALTEK_MODE1}", ATTR{bConfigurationValue}="$env{REALTEK_MODE1}"
+
+# Nvidia
+ATTR{idVendor}=="0955", ATTR{idProduct}=="09ff", ATTR{bConfigurationValue}!="$env{REALTEK_MODE1}", ATTR{bConfigurationValue}="$env{REALTEK_MODE1}"
+
+# LINKSYS
+ATTR{idVendor}=="13b1", ATTR{idProduct}=="0041", ATTR{bConfigurationValue}!="$env{REALTEK_MODE1}", ATTR{bConfigurationValue}="$env{REALTEK_MODE1}"
+
+LABEL="usb_realtek_net_end"
```
Comment (sever-sever) [inline data/live-build-config/includes.chroot/etc/udev/rules.d/50-usb-realtek-net.rules:46]: Need to be careful with this.
One of our customers didn't see Realtek interfaces after this script at all.
Tested on.
```
# lsusb -v
Bus 002 Device 003: ID 0bda:8156 Realtek Semiconductor Corp.
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               3.20
  bDeviceClass            0
  bDeviceSubClass         0
  bDeviceProtocol         0
  bMaxPacketSize0         9
  idVendor           0x0bda Realtek Semiconductor Corp.
  idProduct          0x8156
```

As I understand, the original script is https://github.com/bb-qq/r8152/blob/master/50-usb-realtek-net.rules

Do you have a chance to test it before merging?

**Comment by cuongdt1994** [comment]
I will buy another usb ethernet 2.5Gbps (Chipset: 8156B) and test again

**Comment by cuongdt1994** [comment]
I discovered the following problem related to Realtek RTL8168:

first.
Currently getting the wrong port by name:

Standard is eth0/eth1/eth2/eth3
Currently eth0/eth1/eth3/eth4

2. No receiving ipv6 from ISP

**Pull request closed without merging**